src/parser/percent.ts

```diff
@@ -3,8 +3,8 @@ import { capValue } from '../util/misc/capValue';
 
 const RE_PERCENT = /^(\d+\.\d+)%|(\d+)%$/;
 
-export function isPercent(value: string | null) {
-  return value && RE_PERCENT.test(value);
```

Comment:
checked that all usages are with a defined value

Reply:
can we put `const RE_PERCENT = /^(\d+\.\d+)%|(\d+)%$/;` inside the function `isPercent` ? i m not sure there is reason to keep a regex alive all the time. The parser is not hot code

Reply:
I thought you wanted all regex to be const to be created once.
Can we not hold back on these things for now?